src/oic/oic/provider.py

```diff
@@ -255,6 +255,9 @@ def __init__(
         logout_path="",
         settings: PyoidcSettings = None,
     ):
+        """
+        :param name the iss, which is the URI of the OP.
```

Comment:
I would rather use the `google` style conventions.

Reply:
I will take a look at it, thanks for your advice.